pkg/registry/apis/provisioning/register.go

```diff
@@ -518,6 +561,13 @@ func (b *ProvisioningAPIBuilder) GetPostStartHooks() (map[string]genericapiserve
 				return err
 			}
 
+			// We do not have a local client until *GetPostStartHooks*, so we can delay init for some
```

Comment:
I moved the tester initialization here so we have access to to the configured client via loopback